bokeh/models/sources.py

```diff
@@ -138,23 +137,6 @@ def _data_from_df(df):
             new_data["index"] = index.tolist()
         return new_data
 
-    @classmethod
-    @deprecated("Bokeh 0.9.3", "ColumnDataSource initializer")
-    def from_df(cls, data):
-        """ Create a ``dict`` of columns from a Pandas DataFrame,
-        suitable for creating a ColumnDataSource.
-
-        Args:
-            data (DataFrame) : data to convert
-
-        Returns:
-            dict(str, list)
-
-        """
-        import warnings
-        warnings.warn("Method deprecated in Bokeh 0.9.3")
-        return cls._data_from_df(data)
-
     def to_df(self):
```

Comment:
Can we check if this is actually used anywhere at all? If not, I would suggest perhaps removing it.